packages/pro-js/src/DataRequest.ts

```diff
@@ -12,9 +12,9 @@ interface TourStateDb extends DBSchema {
 }
 
 class DataRequest {
-  apiKey: string;
-  apiPath: string;
-  properties?: { [key: string]: unknown };
+  private apiKey: string;
+  private apiPath: string;
+  private properties?: { [key: string]: unknown };
```

Comment:
The React package was complaining about these being private. Do they need to be? Wondering if using the JS way of making them private would make TS happier? Like `#apiKey` or whatever

Reply:
interesting, I don't have the issue b/c the react package isn't reading them, just passing them through. I agree with following correct syntax though.